docs/en/01_Overview.md

```diff
@@ -0,0 +1,234 @@
+
+```php
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+$publish = new PublishItemsJob(21);
+QueuedJobService::singleton()->queueJob($publish);
+```
+
+To schedule a job to be executed at some point in the future, pass a date through with the call to queueJob
+The following will run the publish job in 1 day's time from now.
+
+```php
+use SilverStripe\ORM\FieldType\DBDatetime;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+$publish = new PublishItemsJob(21);
+QueuedJobService::singleton()
+    ->queueJob($publish, DBDatetime::create()->setValue(DBDatetime::now()->getTimestamp() + 86400)->Rfc2822());
+```
+
+## Immediate Jobs
+
+Jobs can be declare to run "immediately", rather than being queued.
+What that means in practice depends on your configuration.
+
+By default, these jobs are run on PHP shutdown,
+in the same process which queued the job
+(see `QueuedJobService::$use_shutdown_function`).
+Code run after shutdown does not affect the response to the requesting client,
+but does continue to consume resources on the server.
+In a worker-based environment such as Apache or Nginx,
+this can have side effects on request processing.
+
+So while this is the easiest setup (zero config or system dependencies),
+there are more robust alternatives:
+
+- [inotifywait](immediate-jobs#inotifywait): Works based on watching files. Useful for immediate jobs.
+- [lsyncd](immediate-jobs#lsyncd): Alternative based on watching files.
+
+## Logging and Error Reporting
+
+Just like any other code in Silverstripe, jobs can create log entries and errors.
+You should use the global `LoggerInterface` singleton
+as outlined in the [framework docs on error handling](https://docs.silverstripe.org/en/4/developer_guides/debugging/error_handling/).
+
+Any log handlers which are configured within your application
+(e.g. services like Sentry or Raygun) will also pick up logging
+within your jobs, to the reporting level you've specified for them.
+
+Additionally, messages handled through `LoggerInterface`
+as well as an exceptions thrown in a job will be logged
+to the database record for the job in the `QueuedJobDescriptor.SavedJobMessages`
+column. This makes it easier to associate messages to specific job runs,
+particularly when running multiple jobs concurrently.
+
+Immediate jobs run through `ProcessJobQueueTask` will also
+log to stderr and stdout when run through the command-line (incl. cron execution).  
+Queued jobs run this way may not log consistently to stdout and stderr,
+see [Troubleshooting](troubleshooting)
+
+## Default Jobs
+
+Some jobs should always be either running or queued to run, things like data refreshes or periodic clean up jobs, we call these Default Jobs.
+See [Default Jobs](default-jobs) for information on how to
+disable or pause these jobs.
+
+## Job states
+
+It's really useful to understand how job state changes during the job lifespan as it makes troubleshooting easier.
+
+The following shows the whole job lifecycle:
```

Comment:
This used to be an image https://github.com/silverstripe/silverstripe-queuedjobs/blob/6/docs/en/index.md#job-states-job-states

I made it this list of bullets because
- I'm not sure what the correct way to get the image to work on both github and docs site is
- This image looked pretty bad

Am happy to change back to image, if you can provide me the exact expression I need to use to get it to work in both scenarios